assignments/prerequisites/python-programming-advanced.ipynb

```diff
@@ -37,20 +37,23 @@
    ]
```

Comment:
<p>Such code is too naive. We need to find some better code sample. And the code must in a function.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/open-academy/machine-learning/pull/328/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='8'/>